pkg/rancher-desktop/assets/translations/en-us.yaml

```diff
@@ -4784,6 +4784,7 @@ snapshots:
       message: "Please wait while the snapshot operation is active.<br/>This may take some time depending on your machine's resources.<br/>Rancher Desktop will be temporarily unavailable during this operation.<br/><br/>Once the snapshot process is complete, the VM will restart."
     showLogs: Show logs
   info:
+    when: " at {when}"
```

Comment:
Do we need to worry about zh-CN?

Reply:
If we can easily do so, let's do it. But otherwise I would say we should not let this slow us down. We don't even have anyone to translate the docs anymore, so we will remove the links to the non-English version from the docs site soon.

We have never built a localized version of Rancher Desktop, we just maintain the capability to potentially do so. We should continue to do this, as long as it doesn't take significant effort. But if it does, then let's ignore it for now and deal with it when we actually want to release a localized version (which may be never).